home-manager-module.nix

```diff
@@ -35,6 +35,11 @@ in
         default = "*-*-* 00:00:00";
       };
     };
+    podman = mkOption {
```

Comment:
Similarly this doesn't do much, as it's the link into `systemd/user-generators/podman-user-generator` that controls which podman to use, which we are doing in nixos module, or externally in the case of standalone home manager.

We should be able to read `config.virtualisation.podman.package` from `osConfig`, and probably default to `pkgs.podman` if `osConfig` doesn't exist.

Reply:
Ah, right. I forgot about `osConfig`